docs/blockchain/contracts/nft-contract/nft-tables.experimental.md

```diff
@@ -0,0 +1,812 @@
+---
+title: 'NFT Tables'
+order: 1
+
+---
+
+# NFT Tables
+
+## factory.b
+
+-   Table: `factory.b`
+-   Code: `eosio.nft.ft`
+-   Scope: `eosio.nft.ft`
+-   Key: `id`
+
+The table contains uniq factories settings and the operational info.
```

Comment:
Should be
The table contains Uniq factories' settings ...

Reply:
updated